ctapipe/core/tool.py

```diff
@@ -138,11 +150,22 @@ def initialize(self, argv=None):
                 self.load_config_file(self.config_file)
             except Exception as err:
                 raise ToolConfigurationError(f"Couldn't read config file: {err}")
-        self.log.info(f"ctapipe version {self.version_string}")
 
         # ensure command-line takes precedence over config file options:
         self.update_config(self.cli_config)
 
+        log_config = create_log_config(
+            name=self.name,
+            log_level=self.log_level,
+            log_file=self.log_file,
+            log_file_level=self.log_file_level,
+        )
+
+        logging.config.dictConfig(log_config)
```

Comment:
can we add a json or yaml file for the log config and only add the default config if none was given?

Reply:
to use with e.g. `--log-config`? I initially decided against this, because the logger options can be set with `--config`

Reply:
Can they? Like configuring each logger? So can I set `ctapipe.io` to DEBUG while the rest is `WARNING`? That's possible with the logging config files.

Reply:
Ah I understand now the direction. Yes I think that's not possible with the config file and would be easier/better using logconfig files